backend/services/deviceconnect/api/http/router.go

```diff
@@ -61,6 +61,7 @@ const (
 type RouterConfig struct {
 	GracefulShutdownTimeout time.Duration
 	MaxRequestSize          int64
+	MaxFileSize             int64
```

Comment:
could you add // comments here to explain what is the difference between these two limits? first look and one can think that PUT request is just a request so why do we need a separate limit?